src/main-process/atom-window.coffee

```diff
@@ -40,9 +40,15 @@ class AtomWindow
     if process.platform is 'linux'
       options.icon = @constructor.iconPath
 
+    if @applyTitleBarSetting()
+      options.titleBarStyle = 'hidden'
+
     @browserWindow = new BrowserWindow options
     global.atomApplication.addWindow(this)
 
+    if @applyTitleBarSetting()
+      @browserWindow.setSheetOffset(23)
```

Comment:
This hard-coded value makes me nervous. If I change the height for the title bar, this will be off. Is there a way to calculate this and update it?